trainee_template/attributes.amlg

```diff
@@ -454,6 +454,9 @@
 		(assign_to_entities (assoc
 			featureAttributes features
 
+			trainedFeatures (sort (indices features))
```

Comment:
won't this overwrite whatever existing treanedFeatures that were automatically added previously? i.e., say you trained some cases first, and then remember to set attributes to specify a bounds for one of them.  won't this blow away all those others that were added?

Reply:
if you trained the features first, they should be in the feature attributes at this point. Then setting the bounds will not alter the indices, so re-setting trainedFeatures as the sorted indices wouldn't lose any features.

Unless I misunderstand your point, I don't think this would be an issue.